client/js/components/statement/splitStatement/CardPaneCard.vue

```diff
@@ -58,13 +58,12 @@
       :text="Translator.trans('segment.edit')"
       @click="$emit('edit-segment', segment.id)" />
     <addon-wrapper
-      :addon-props="{
-        segmentStatus: segment.status
-      }"
+      v-if="segment.status !== 'confirmed'"
```

Comment:
This would mean addon mounting here is only ever allowed for non-confirmed segments. Is this what we want?